api/internal/impl/cipher/decrypt.go

```diff
@@ -40,47 +43,54 @@ import (
 //	defer source.Close()
 //
 //	reader := bytes.NewReader(encryptedData)
-//	plaintext, err := DecryptStream(source, reader, "application/octet-stream")
+//	plaintext, err := DecryptStream(source, reader)
```

Comment:
No need to give content-type as an argument since stream encrption/decryption always reads bytes; so the only thing it understands is octet streams anyway.

same counter-argument applies for the json variants. they don't understand octet, they only know application/json, so they assume application/json as content type.